spec/beaker-pe/install/pe_utils_spec.rb

```diff
@@ -1027,6 +1027,7 @@ def slice_installer_options(host)
     let(:puppetdb) { make_host('puppetdb', :pe_ver => '2016.4', :roles => [ 'database' ]) }
     let(:console) { make_host('console', :pe_ver => '2016.4', :roles => [ 'dashboard' ]) }
     let(:agent) { make_host('agent', :pe_ver => '2016.4', :roles => ['frictionless']) }
+    let(:pe_postgres) { make_host('puppetdb', :pe_ver => '2016.4', :roles => [ 'pe_postgres' ]) }
```

Comment:
This is a little confusing, as there is already a host mocked out/named `puppetdb` with a different role. Should this also be `pe_postgres`, or is there a reason it is also `puppetdb`?

Reply:
Good catch